content/z.yml

```
@@ -116,8 +116,8 @@ tasks:
             - @(if drop && !table && !where) echo "Dropping database <comment>$(database)</comment>"; echo "Creating database <comment>$(database)</comment>";
             - @(if drop && !table && !where) mysql $(defaults_local ? sprintf("--defaults-file=%s", defaults_local)) -e "DROP DATABASE IF EXISTS $(database); CREATE DATABASE $(database);"
             - @(if !stdout) echo "Pull database from <info>$(target_env)</info> into <comment>$(database)</comment>"
-            - @(if !stdout) $(!content.is_local() ? content.fmt.ssh.prefix(target_env))mysqldump $(content.fmt.defaults.remote) --opt -Q$(VERBOSE ? "v") $(where ? sprintf("--where='%s'", where)) $(envs[target_env].db) $(table ? table) $(!content.is_local() ? content.fmt.ssh.suffix(target_env)) | mysql $(defaults_local ? sprintf("--defaults-file=%s", defaults_local)) $(database)
-            - @(if stdout)  $(!content.is_local() ? content.fmt.ssh.prefix(target_env))mysqldump $(content.fmt.defaults.remote) --opt -Q$(VERBOSE ? "v") $(where ? sprintf("--where='%s'", where)) $(envs[target_env].db) $(table ? table) $(!content.is_local() ? content.fmt.ssh.suffix(target_env))
+            - @(if !stdout) $(!content.is_local() ? content.fmt.ssh.prefix(target_env))mysqldump $(content.fmt.defaults.remote) --opt -Q --no-tablespaces$(VERBOSE ? " -v")$(where ? sprintf(" --where='%s'", where)) $(envs[target_env].db) $(table ? table) $(!content.is_local() ? content.fmt.ssh.suffix(target_env)) | mysql $(defaults_local ? sprintf("--defaults-file=%s", defaults_local)) $(database)
+            - @(if stdout)  $(!content.is_local() ? content.fmt.ssh.prefix(target_env))mysqldump $(content.fmt.defaults.remote) --opt -Q --no-tablespaces$(VERBOSE ? " -v")$(where ? sprintf(" --where='%s'", where)) $(envs[target_env].db) $(table ? table) $(!content.is_local() ? content.fmt.ssh.suffix(target_env))
```

```suggestion
            - @(if !stdout) $(!content.is_local() ? content.fmt.ssh.prefix(target_env))mysqldump $(content.fmt.defaults.remote) --opt -Q --no-tablespaces$(VERBOSE ? " -v")$(where ? sprintf(" --where='%s'", where)) $(envs[target_env].db) $(table ? table) $(!content.is_local() ? content.fmt.ssh.suffix(target_env)) | mysql $(defaults_local ? sprintf("--defaults-file=%s", defaults_local)) $(database)
            - @(if stdout)  $(!content.is_local() ? content.fmt.ssh.prefix(target_env))mysqldump $(content.fmt.defaults.remote) --opt -Q --no-tablespaces $(VERBOSE ? "-v") $(where ? sprintf("--where='%s'", where)) $(envs[target_env].db) $(table ? table) $(!content.is_local() ? content.fmt.ssh.suffix(target_env))
```

(flavor) liever altijd tussen alle argumenten een laten spatie, ipv de patie onderdeel maken van de if/else.  Dit maakt het minder foutgevoelig en de code leesbaarder.  Voor de shell interpreter maakt het niet uit dat er dan soms meerdere spaties tussen argumenten staan.

---

(flavor) ik vind het lelijk als er allemaal loze spaties in het command komen als ik het moet kopiëren en plakken om iets handmatig te gaan doen.

🤔 jouw argumenten zijn alleen wel wat sterker dan mijn esthetische argument 😆 

Ik zat wel te denken... moeten we dit niet gewoon in een functie zetten in de `Plugin.php`? Dan kun je een array van argumenten samenstellen en die imploden met een spatie ertussen. Dat maakt het veel duidelijker. Nu wordt het allemaal zo lang... (en het staat er ook nog eens dubbel in). Leon had ook nog een fix met een extra argument...

---

>  jouw argumenten zijn alleen wel wat sterker dan mijn esthetische argument 

Als mijn argumenten sterker zijn dan jou esthetische argument, dan had dit dus niet zo gemerged moeten worden... @7ochem wil je dit dan alsnog aanpassen?  (daarmee bedoel ik de 'suggested change' van drie dagen geleden)